src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs

```diff
@@ -146,13 +146,17 @@ public async Task<IReadOnlyCollection<ResourceChangeData>> GetRecordsAsync(long
                     case SqlErrorCodes.TimeoutExpired:
                         throw new TimeoutException(ex.Message, ex);
                     default:
+#pragma warning disable CA2254 // Template should be a static expression
```

Comment:
Should we be disabling this warning globally?

Reply:
I personally like that warning, and I try to update most of the places to conform with it.
Right now there is 11 exceptions for the rule in my PR. Some of them is we write exception into log, and they cannot be static. 
Some of them string is actually static, but we either take it from resource file or compose once (because of length of string)

Reply:
And I reduce it to 1 exception case,